article/app/views/fragments/recipeArticleBody.scala.html

```diff
@@ -1,14 +1,14 @@
 @import layout.ContentWidths.MainMedia
```

Comment:
You are still using immersive [media widths](https://github.com/guardian/frontend/blob/master/common/app/layout/WidthsByBreakpoint.scala#L185). These are needlessly big on mobile. Would you be able to change this soon?